server/internal/cmd/opp/internal/safetensors/safetensors.go

```diff
@@ -95,7 +97,7 @@ func (m *Model) readTensors(fname string) ([]*Tensor, error) {
 
 	tt := make([]*Tensor, 0, len(raws))
 	for name, raw := range raws {
-		if !strings.HasPrefix(name, "model.layer") {
+		if !strings.HasPrefix(name, "model.") {
```

Comment:
This will miss the following (e.g. model-00004-of-00004.safetensors in Meta-Llama-3.1-8B-Instruct
```json
  "lm_head.weight": {
    "dtype": "BF16",
    "shape": [
      128256,
      4096
    ],
    "data_offsets": [
      0,
      1050673152
    ]
  },
```

Reply:
Okay. I'll just skip the manifest only.